plugins/coral-plugin-comment-content/client/components/CommentContent.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Linkify from 'react-linkify';
+
+const name = 'coral-plugin-comment-content';
+
+const CommentContent = ({comment}) => {
+  const textbreaks = comment.body.split('\n');
+  return <div className={`${name}-text`}>
+    {
+      textbreaks.map((line, i) => {
+        return (
+          <span key={i} className={`${name}-line`}>
+            <Linkify>
```

Comment:
Missing something like `properties={{target: '_parent'}}`, so the links work inside the iframe.